notebook/templates/notebook.html

```diff
@@ -104,16 +104,17 @@
                         </li>
                         <li class="divider"></li>
                         <li id="print_preview"><a href="#">{% trans %}Print Preview{% endtrans %}</a></li>
+                        <li id="custom_download"><a href="#">{% trans %}Custom Download {% endtrans %}</a></li>
                         <li class="dropdown-submenu"><a href="#">{% trans %}Download as{% endtrans %}</a>
                             <ul id="download_menu" class="dropdown-menu">
-                                <li id="download_ipynb"><a href="#">{% trans %}Notebook (.ipynb){% endtrans %}</a></li>
-                                <li id="download_script"><a href="#">{% trans %}Script{% endtrans %}</a></li>
-                                <li id="download_html"><a href="#">{% trans %}HTML (.html){% endtrans %}</a></li>
-                                <li id="download_slides"><a href="#">{% trans %}Reveal.js slides (.html){% endtrans %}</a></li>
-                                <li id="download_markdown"><a href="#">{% trans %}Markdown (.md){% endtrans %}</a></li>
-                                <li id="download_rst"><a href="#">{% trans %}reST (.rst){% endtrans %}</a></li>
-                                <li id="download_latex"><a href="#">{% trans %}LaTeX (.tex){% endtrans %}</a></li>
-                                <li id="download_pdf"><a href="#">{% trans %}PDF via LaTeX (.pdf){% endtrans %}</a></li>
```

Comment:
Can I ask you why you remove `{% trans %}` on these ones?

Reply:
It was a while ago that I had made that decision with input from either @takluyver or @Carreau, on the grounds that for some of those, a translation would never make sense (e.g., LaTeX should always be LaTeX). However you’re right that we can always include the translation tag and assume that there won’t be a mistranslation (e.g., treating reST as rest (the verb) with weird capitalisation and translating accordingly). Certainly Notebook, Script and the word “via” make sense to translate. I can add those tags back.